lib/index.js

```diff
@@ -509,6 +511,20 @@ class KuzzleCluster {
     this.node.state.locks.delete.delete(request.input.body.roomId);
   }
 
+  resetSecurityCache () {
+    this.node.broadcast('cluster:admin:resetSecurity');
+  }
+
+  dump (request) {
+    const suffix = request.input.args.suffix ? request.input.args.suffix : '';
+
+    this.node.broadcast('cluster:admin:dump', { suffix });
+  }
+
+  shutdown () {
+    this.node.broadcast('cluster:admin:shutdown')
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)